### PR TITLE
Fix errors in 'make' cmd

### PR DESCRIPTION
**Pull request by v01dstar**

### CODE REVIEW

File: tools/db_bench_tool.cc [hunk 1]
```diff
@@ -5094,7 +5092,7 @@ class Benchmark {
             seek++;
             read++;
             if (single_iter->Valid() && single_iter->key().compare(key) == 0) {
-              seek_found++;
+              found++;
```
Comment (Connor1996) [inline tools/db_bench_tool.cc:5095]: why

File: CMakeLists.txt [hunk 2]
```diff
@@ -177,19 +177,19 @@ if (WITH_TITAN_TOOLS)
...
   target_link_libraries(titandb_stress ${TOOLS_LIBS})
 
   add_executable(titandb_bench tools/db_bench.cc tools/db_bench_tool.cc)
-  target_include_directories(titandb_bench PRIVATE ${gflags_INCLUDE_DIR})
+  target_include_directories(titandb_bench PRIVATE ${GFLAGS_INCLUDE_DIR})
   target_link_libraries(titandb_bench ${TOOLS_LIBS})
 
   add_executable(titan_manifest_dump tools/manifest_dump.cc)
-  target_include_directories(titan_manifest_dump PRIVATE ${gflags_INCLUDE_DIR})
+  target_include_directories(titan_manifest_dump PRIVATE ${GFLAGS_INCLUDE_DIR})
```
Comment (Connor1996) [inline CMakeLists.txt:188]: does it matter?

File: tools/db_bench_tool.cc (hunk 1 above)
Comment (v01dstar) [reply tools/db_bench_tool.cc:5095]: `seek_found` is never used. From line 5117, I think the intent of the original author was to use them interchangeably .

File: CMakeLists.txt (hunk 2 above)
Comment (v01dstar) [reply CMakeLists.txt:188]: It matters in my setup. 

cmake version 3.23.2

**Comment by Connor1996** [comment]
Closed as included in #298

**Pull request closed without merging**